### ISSUE TITLE
Remove global config object

### ISSUE THREAD

**Issue by fluffy-critter** [PlaidWeb/Publ#113]
## Expected Behavior

Get rid of `publ/config.py` and instead store the config on the Publ object itself, ideally reusing Flask configuration wherever appropriate.

## Current Behavior

Global config bad. App object-level config good.

## Context

Make this a better citizen in the Flask ecosystem.

**Comment by fluffy-critter** [comment]
Well, I did all the work to separate stuff out and put the Publ configuration in the app object, but then I ran into some weirdness where the request handler function mapped to `None` even though every step in the debugger did the right thing. Weird.

Anyway, it turns out there's only so much decoupling that can happen since Peewee still has its global configuration for the database itself. And I'm not sure how valuable this would be anyway.

See branch https://github.com/fluffy-critter/Publ/tree/feature/113-kill-global-config for my attempt at doing this, likely to be abandoned.

**Comment by fluffy-critter** [comment]
The PonyORM migration only makes this even more pointless, because PonyORM directly binds the ORM mapping to the database at a global level. I'm comfortable just closing this as invalid.

**Comment by fluffy-critter** [comment]
I still have an itch about this so I'm reopening it, especially after the impending completion of #205

**Comment by fluffy-critter** [comment]
This would still be useful for improving the unit tests.

**Comment by fluffy-critter** [comment]
> The PonyORM migration only makes this even more pointless, because PonyORM directly binds the ORM mapping to the database at a global level. I'm comfortable just closing this as invalid.

ponyorm has a pattern for this, which is incredibly obvious in hindsight: https://github.com/ponyorm/pony/issues/330#issuecomment-470153594

tl;dr: encapsulate `model`'s classes in a closure, something like:

```python
def build_model():
    db = orm.Database()
    class GlobalConfig(db.Entity):
        #...

    class Model:
        def __init__(self):
            #pylint:disable=invalid-name
            self.GlobalConfig = GlobalConfig
            # ....
        def setup(self):
            #...
        def reset(self):
            #...
    return Model()
```

And then the Model object which is returned gets attached to the Publ instance, which might also solve #134 since the model will persist as long as the app does.
